src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -51,6 +51,7 @@ public function setup() {
 
 		add_filter( 'tribe_events_views_v2_view_template_vars', [ $this, 'filter_widget_template_vars' ], 20, 2 );
 		add_filter( "tribe_events_views_v2_view{$this->view_slug}_template_vars", [ $this, 'filter_widget_template_vars' ], 20, 2 );
+		add_action( 'tribe_events_views_v2_widget_after_enqueue_assets', [ $this, 'action_enqueue_manager' ], 10, 3 );
```

Comment:
Either here or in the method long description, could you add an explanation of why this is done? 
It's not apparent from the code.

Reply:
For our widgets, when we enqueue JS files, we enqueue the viewport, breakpoints, and manager. After that, we add the assets for the specific widget. However, if we have a JS file (as in the countdown widget), we need the countdown widget JS to be enqueued before the manager (priority does not seem to take effect when force enqueuing). This also becomes a problem when we have multiple widgets, all with their own JS. So this was my solution to the problem.

Reply:
I understand the solution, I just would like the quick description noted in the code to avoid future-dev come in, see a clunky code, and remove it.

Reply:
OK, will add.

Reply:
If the manager is always enqueued, why not use it as a dependency of the widget js - then manager has to be loaded first?

Reply:
The manager needs to be loaded last, as it triggers events that other JS files use.